6_temp_wqp_fetch/out/temperature_180814233303_001.feather.ind

```diff
@@ -0,0 +1,3 @@
+warning: dry_put=TRUE; not actually pushed
```

Comment:
Yeah, that's my plan. I added that `warning` line so that reviewers like you could notice when pull requesters like me accidentally failed to switch over to dry_put=FALSE before finalizing the PR - it's a good thing to point out. In this particular case I'd like to merge without updating all these files because they're just tests, so next time I do a pull I want to delete these files and replace them with the real deal for a real site list. But if my next PR comes through with this warning, please do point it out.